### DISCUSSION TITLE
Is Packet Capture fixed in 6.35 or not?

### DISCUSSION THREAD

**Discussion by Routhinator** [ffxiv-teamcraft/ffxiv-teamcraft#2545] [Q&A] [answered]
Hey folks, there was a banner after the 6.35 update that said packet capture would be fixed in an upcoming version. New version dropped yesterday and the banner is gone, but packet capture is still not working. I've tried reinstalling everything and went through all the troubleshooting steps.. and no dice.

If it's still not fixed, which issue is tracking this?

**Comment by Supamiu** [comment, marked as answer]
Hi, there's no issue tracking this, please prefer Discord for this as it's reaching more people 😄 

Packet capture for 6.35 is being worked on and will require a bit more time, more info on Discord's known-bugs and updates channels: https://discord.gg/r6qxt6P
